core/storage/question/gae_models_test.py

```diff
@@ -27,17 +27,34 @@
 class QuestionModelUnitTests(test_utils.GenericTestBase):
     """Tests the QuestionModel class."""
 
-    def test_create_question(self):
+    def test_create_question_empty_skill_id_list(self):
         state = state_domain.State.create_default_state('ABC')
         question_state_data = state.to_dict()
         language_code = 'en'
         version = 1
         question_model = question_models.QuestionModel.create(
-            question_state_data, language_code, version)
+            question_state_data, language_code, version, [])
 
         self.assertEqual(
             question_model.question_state_data, question_state_data)
         self.assertEqual(question_model.language_code, language_code)
+        self.assertItemsEqual(question_model.linked_skill_ids, [])
```

Comment:
Ignore the comment above. In gae_models, you store it as a set. Then here it should be set() instead of [].

Reply:
Google app engine converts it to a list itself.